app/templates/challenge.html

```diff
@@ -8,216 +8,244 @@
+                    {% endblock %}
+                  </li>
+                </ul>
               </div>
+              {% endif %}
             </div>
-
           </div>
-
         </div>
 
-        <p style="margin-left: 20px;">Created on {{ challenge.created }}</p>
       </div>
-      <div class="col-3" style="margin-left: 16px;">
-        <div style="position: absolute;" id="social-float">
-          
 
-          <div class="modal modal-animated--zoom-in" id="share-modal">
-            <a href="#searchModalDialog" class="modal-overlay close-btn" aria-label="Close"></a>
-            <div class="modal-content" style="min-width: 500px; border-radius: 16px;">
-              <div class="modal-body">
-              {% if challenge.image %}
-                <div class="share-image-preview" style="background-image: url({{ challenge.image.url }})"></div>
-              {% else %}
-                <div class="share-image-preview" style="background-image: url({{ static('img/polar-bear-2.png') }})"></div>
-              {% endif %}
-              <h3 class="u-text-center">Share This Challenge</h3>
-                <div class="row">
-                  <a class="popup clipboard-btn share-social-icon level-content level-item" onclick="displayLinkCopied('1')">
-                    <i class="fas fa-link fa-lg" style="color: #868686;"></i>
-                    <span class="popuptext" id="copy-popup-1">Link copied!</span>
-                  </a>
-                  <a onclick="shareOnFB(window.location.href.toString())" class="share-social-icon level-content level-item">
-                    <i class="fab fa-facebook fa-lg" aria-hidden="true"></i>
-                  </a>
-                  <a onclick="shareOnTwitter(window.location.href.toString())" class="share-social-icon level-content level-item">
-                    <i class="fab fa-twitter fa-lg"></i>
-                  </a>
-                  <a onclick="shareOnLinkedIn(window.location.href.toString())" class="share-social-icon level-content level-item">
-                    <i class="fab fa-linkedin fa-lg"></i>
-                  </a>
-                  <a onclick="shareOnReddit(window.location.href.toString())" class="share-social-icon level-content level-item">
-                    <i class="fab fa-reddit fa-lg"></i>
-                  </a>
-                </div>
-              </div>
-            </div>
-          </div>
-          {% if challenge.is_open() %}
-            <a href="{{ url('project_challenge_create', pk=challenge.pk) }}" class="u-center">
-              <button class="btn-primary btn--pilled" style="width: 100%;">Create Solution</button>
-            </a>
-          <a href="#share-modal" class="font-bold u-center" style="margin: 20px 0;">Share Challenge &nbsp;<i class="fas fa-share"></i></a>
-          {% endif %}
+    </div>
 
+    <p style="margin-left: 20px;">Created on {{ challenge.created }}</p>
+  </div>
+  <div class="col-3" style="margin-left: 16px;">
```

Comment:
This block should be nested inside of `<div class="row challenge-wrap">`
Otherwise, it appears at the bottom of the challenge instead of floating to the right of the challenge.
Structure should look like this:
```html
<div class="row challenge-wrap">
  <div class="col-8">
      <div class="challenge-card">...</div>
      <p style="margin-left: 20px;">Created on 2021-03-20</p></div>
  </div>
  <div class="col-3" style="margin-left: 16px;">...</div>
</div>
```